### PR TITLE
Get and set revisions limit per database

### PR DESCRIPTION
**Pull request by c0nstantx**

### CODE REVIEW

File: lib/cradle/database/index.js [hunk 1]
```diff
@@ -47,6 +47,30 @@ Database.prototype.create = function (callback) {
...
+                return callback && callback(true, null);
+            } else {
+                return callback && callback(null, parseInt(res.body));
+            }
+        });
+    } else {
+        this.query({
+            method: 'PUT',
+            path: '_revs_limit',
+            body: !isNaN(revisions) ? parseInt(revisions) : 1000
```
Comment (panuhorsmalahti) [inline lib/cradle/database/index.js]: Should use ``Number.isNaN``. Also, ``parseInt`` should specify radix.

**Comment by panuhorsmalahti** [comment]
Looks great! Can you add a short mention in the documentation?

**Comment by c0nstantx** [comment]
Yes, I will make the changes u noted on the code and add to documentation.

**Comment by panuhorsmalahti** [comment]
Thanks 👍

**Pull request merged**